package/autoyast2.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Jul 18 15:05:10 UTC 2019 - Jan Engelhardt <jengelh@inai.de>
+
+- Replace old $RPM_* shell vars
```

Comment:
I'd remove this line, I cannot see any related change in the diff.

Reply:
yep, as it was done already by lcp. So I will remove it